client-app/core/api/graphql/settings/queries/getStoreSettings/getStoreSettings.graphql

```diff
@@ -0,0 +1,36 @@
+#import "../../../fragments/allCurrencyFields.graphql"
+#import "../../../fragments/allLanguageFields.graphql"
+
+query GetStoreSettings($storeId: String!) {
+  store(storeId: $storeId) {
+    storeId
+    storeName
+    catalogId
+    storeUrl
+    userName
+    userId
+    defaultLanguage {
+      ...allLanguageFields
+    }
+    availableLanguages {
+      ...allLanguageFields
+    }
+    defaultCurrency {
+      ...allCurrencyFields
+    }
+    availableCurrencies {
+      ...allCurrencyFields
+    }
+    settings {
+      quotesEnabled
+      subscriptionEnabled
+      taxCalculationEnabled
+      anonymousUsersAllowed
+      isSpa
+      emailVerificationEnabled
+      emailVerificationRequired
+      createAnonymousOrderEnabled
+      seoLinkType
+    }
```

Comment:
We use `anonymousUsersAllowed` only (as `anonymousAccessEnabled`)

Reply:
New settings set. Added as a marker for theme_settings / store_settings confuse